mix.exs

```diff
@@ -11,7 +11,7 @@ defmodule EredisCluster.Mixfile do
       description: "An erlang wrapper for eredis library to support cluster mode",
       docs: [
         main: "readme",
-        extras: ["README.md", "doc/eredis_cluster.md", "doc/eredis_cluster_monitor.md"],
+        extras: ["README.md", "CHANGELOG.md", "doc/eredis_cluster.md", "doc/eredis_cluster_monitor.md"],
```

Comment:
We probably should add the CHANGELOG.md to the Elixir package aswell in `defp package do` below, since the default values for which files to includes is [according to hex.pm](https://hex.pm/docs/publish)

      files: ~w(lib priv .formatter.exs mix.exs README* readme* LICENSE*
                license* CHANGELOG* changelog* src),

Reply:
Thanks. Done.